### DISCUSSION TITLE
Software Screen Capture keeps changing device

### DISCUSSION THREAD

**Discussion by Nidgy** [awawa-dev/HyperHDR#581] [Q&A]
Hi there, I use HyperHDR on on my PC and works like a charm. However, I am using two monitors and have a LED strip attached to my main monitor. I do not know if it is a problem withing my Windows-, NVIDIA- or HyperHDR settings. Sorry for that!

But my screen numbers keep changing from 1 to 2, selecting automatic also works random. So every time I power on my main monitor, the Screen Grabber will record my main or second monitor. My main monitor is a LG OLED TV, so maybe that has something to do with the settings? I hope someone can help me out with this problem, thank you in advance :) !

![image](https://github.com/awawa-dev/HyperHDR/assets/100786666/c0568b2e-754e-4f68-9d24-e394db0ef0b5)

**Comment by awawa-dev** [comment]
Hi
In display properties:
![obraz](https://github.com/awawa-dev/HyperHDR/assets/69086569/ad5063c0-69e0-4a5e-9aa0-aeeaf0d1dab3)
identify your displays when the system is working correctly e.g. monitor is 1 and LG is 2.
If the problem occurs again, repeat the procedure to see if the display indexes (or anything else) have changed.
I suspect the gfx drivers, although I don't use software capturing, my Windows sometimes rearranges them depending on display power-on order.

**Reply by Nidgy** [reply]
Thanks for your reply! Windows is always using the same monitor numbers. 1 for my second screen and 2 for my LG.  So I think you are right and it has to do something with my gfx drivers. Looks like NVIDIA control panel changes them, going further into this problem :( !
